### PR TITLE
Add option to specify start of the week

### PR DESCRIPTION
**Pull request by pedrorivotti**
This PR adds a configurable option, `options.week_start`, that allows specifying the start day of a week for reporting and logging purposes. 

This is useful for those who (like me) need to report weekly time based on a starting day other than Monday. Sunday would be the other common starting day, but I've added the option to specify any day of the week, for completeness. 

I've included tests, and a short entry in the docs.

Open to suggestions, of course!

### CODE REVIEW

**Comment by jmaupetit** [comment]
Hi @pedrorivotti ! Thanks for this useful feature :pray: We'll review it quickly.

File: watson/cli.py [hunk 1]
```diff
@@ -57,6 +57,13 @@ def convert(self, value, param, ctx):
             # expected by the user, so that midnight is midnight in the local
             # timezone, not in UTC. Cf issue #16.
             date.tzinfo = tz.tzlocal()
+
+            # Add an offset to match the week beginning specified in the
+            # configuration
+            if param.name == "week":
+                week_start = ctx.obj.config.get("options", "week_start")
```
Comment (k4nar) [inline watson/cli.py]: Shouldn't we set a default as well?

Comment (pedrorivotti) [reply watson/cli.py]: Thanks, @k4nar , well spotted!

**Comment by k4nar** [comment]
Thanks!

**Pull request merged**